src/temaWeek6LocalStore/main/LocalStore.java

```diff
@@ -0,0 +1,258 @@
+package temaWeek6LocalStore.main;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class LocalStore {
+	private String name;
+	private double price;
+	private String validityDate;
+	private double weight;
+	private String ID;
+	private String storageTemp;
+	private String vitamin;
+	private int quantity;
+	private String saleDate;
+	private int inputQuantity;
+	private Product prod;
+	
+	List<StockItem> stockItems=new ArrayList<>();
+	List<SoldItem> soldItems=new ArrayList<>();
+	
+	
+	
+	Product p=new Product(name,price,validityDate,weight,ID);
+	StockItem stock=new StockItem(new Animal(name,price, validityDate,weight, ID,storageTemp),quantity);
+	StockItem sold=new StockItem(new Animal(name,price, validityDate,weight, ID,storageTemp),quantity);
+	StockItem vegetableItem=new StockItem(new Vegetable(name,price, validityDate,weight, ID,vitamin),quantity);
+	
+	
+	//	menu
+	public void getMenu() {
+		boolean running = true;
+		
+		System.out.println("Welcome to LocalStore!");
+		while (running) {
+			System.out.println("\n Enter your choice: " +//options in the menu
+					"\n 1.Create product and add it to stock" +
+					"\n 2.Sell products" +
+					"\n 3.Display daily sales report" +
+					"\n 4.Exit");
+			
+			
+			Scanner scan = new Scanner(System.in);
+			String option = scan.next();
+			LocalStore lc = new LocalStore();
+			switch (option) {
+				case "1": {
+					System.out.println("Product type (1 - animal; 2 - vegetable):");
+					String type = scan.next();
+					if (type.equals("1")) {//adds an animal product to stockItem
+						
+						lc.addToStockAnimal(stockItems);
+						break;
+					}
+					if (type.equals("2")) {//adds an vegetable product to stockItem
+						lc.addToStockVegetable(stockItems);
+						break;
+					}
+					if(!type.equals("1")||!type.equals("2")){
+						System.out.println("Enter 1 or 2!");
+						break;
+					}
+				}
+				case "2": {//sell products based on input
+					System.out.println("Items in stock: \n"+stockItems);
+					System.out.println("Enter ID ");
+					String inputID = scan.next();
+					lc.setInputQuantity(inputQuantity);
+					inputQuantity=lc.getInputQuantity();
+					lc.addSoldItems( stockItems,soldItems,inputID,inputQuantity);//adds sold items to soldItem list
+					lc.sellProduct(stockItems,inputID,inputQuantity);
+					//updates the list of stockItems based on input
+					System.out.println("");
+					System.out.println("Remaining items in stock: \n"+stockItems);//print remaining items in the stockList
+					break;
+					
+				}
+				case "3": {
+					System.out.println("");
+					printSoldItems(soldItems);//print soldItem list
+					break;
+				}
+				case "4": {
+					running = false;
+				}
+			}
+			
+		}
+		
+	}
+	//	method created a stockItem of type animal
+	public StockItem createAnimalProduct() {
+		Animal a=new Animal(name,price, validityDate,weight, ID,storageTemp);
```

Comment:
all those values have empty or old values... why not just create a blank condtructor if you're calling the setter methods below?
Animal a = new Animal();